tests/unit/test_storage_factory.py

```
@@ -0,0 +1,221 @@
+
+
+class TestPostgresBackend:
+    async def test_postgres_returns_postgres_adapter(self, tmp_path: Path) -> None:
+        """STORAGE_BACKEND=postgres → PostgresStorageAdapter が返される."""
+        settings = make_settings(
+            storage_backend="postgres",
+            postgres_host="localhost",
+            postgres_port=5432,
+            postgres_db="test_db",
+            postgres_user="test_user",
+            postgres_password="secret",
```

Ruff (0.15.7)

[error] Possible hardcoded password assigned to argument: "postgres_password"

(S106)